dbld/rules rpm-centos-7 fails due loggen library path changes

# syslog-ng
## Version of syslog-ng
Fresh git snapshot: syslog-ng-3.24.1.176.g39ed128

## Platform
CentOS 7 platform within DBLD

# Issue
## Failure
```dbld/rules rpm-centos-7```
fails with the following error:
```
RPM build errors:
    Directory not found: /root/rpmbuild/BUILDROOT/syslog-ng-3.24.1.176.g39ed128-1+20191120T122658.el7.x86_64/usr/lib64/syslog-ng/loggen
    File not found by glob: /root/rpmbuild/BUILDROOT/syslog-ng-3.24.1.176.g39ed128-1+20191120T122658.el7.x86_64/usr/lib64/syslog-ng/loggen/libloggen*
make: *** [dbld/rules:118: rpm-centos-7] Error 1
```
When removed the related two lines from the spec file, I get:
```
    Installed (but unpackaged) file(s) found:
   /usr/lib/syslog-ng/loggen/libloggen_socket_plugin.so
   /usr/lib/syslog-ng/loggen/libloggen_ssl_plugin.so
```

Before modifying any spec files: is this an expected change?

--- a/configure.ac
+++ b/configure.ac
@@ -161,6 +161,10 @@ AC_ARG_WITH(module-dir,
    [  --with-module-dir=path   Use path as the directory to install modules into],
    moduledir=$with_module_dir)
 
+AC_ARG_WITH(loggen-plugin-dir,
+   [  --with-loggen-plugin-dir=path   Use path as the directory to install loggen plugins into],
+   loggenplugindir=$with_loggen_plugin_dir)
+
 AC_ARG_WITH(module-path,
    [  --with-module-path=path   Use path as the list of ':' separated directories looked up when searching for modules],
    module_path=$with_module_path)

--- a/configure.ac
+++ b/configure.ac
@@ -130,7 +130,7 @@ if test "x$exec_prefix" = "xNONE"; then
 fi
 pidfiledir='${localstatedir}'
 moduledir='${exec_prefix}/lib/syslog-ng'
-loggenplugindir='${exec_prefix}/lib/syslog-ng/loggen'
+loggenplugindir='${moduledir}/loggen'
 toolsdir='${datadir}/syslog-ng/tools'
 xsddir='${datadir}/syslog-ng/xsd'
 

--- a/lib/reloc.c
+++ b/lib/reloc.c
@@ -58,6 +58,7 @@ path_resolver_populate_configure_variables(PathResolver *self, const gchar *sysp
   path_resolver_add_configure_variable(&self->super, "${datarootdir}", SYSLOG_NG_PATH_DATAROOTDIR);
   path_resolver_add_configure_variable(&self->super, "${datadir}", SYSLOG_NG_PATH_DATADIR);
   path_resolver_add_configure_variable(&self->super, "${localstatedir}", SYSLOG_NG_PATH_LOCALSTATEDIR);
+  path_resolver_add_configure_variable(&self->super, "${moduledir}", SYSLOG_NG_PATH_MODULEDIR);
 }
 
 

